frontend/apps/marketing/src/contentful/registration/code.org/index.ts

```diff
@@ -187,6 +190,10 @@ const contentfulRegistration = {
       component: Link,
       definition: LinkContentfulComponentDefinition,
     },
+    {
+      component: ActionBlockCollection,
```

Comment:
Do we need to copy over to CSForAll registration as well?

Reply:
Yes, good call! Updated.